samples/notebooks/dotnet/7-DALL-E-2.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set up the environment, importing Semantic Kernel SDK and SkiaSharp, used to display images inline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    },
+    "tags": [],
+    "vscode": {
+     "languageId": "polyglot-notebook"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><div></div><div></div><div><strong>Installed Packages</strong><ul><li><span>Microsoft.SemanticKernel, 0.9.61.1-preview</span></li><li><span>SkiaSharp, 2.88.3</span></li></ul></div></div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#r \"nuget: Microsoft.SemanticKernel, 0.9.61.1-preview\"\n",
+    "#r \"nuget: SkiaSharp\"\n",
+    "\n",
+    "#!import config/Settings.cs\n",
+    "#!import config/Utils.cs\n",
+    "#!import config/SkiaUtils.cs\n",
+    "\n",
+    "using Microsoft.SemanticKernel;\n",
+    "using Microsoft.SemanticKernel.Configuration;\n",
+    "using Microsoft.SemanticKernel.Orchestration;\n",
+    "using Microsoft.SemanticKernel.KernelExtensions;\n",
+    "using Microsoft.SemanticKernel.AI.ImageGeneration;\n",
+    "using Microsoft.SemanticKernel.AI.Embeddings;\n",
+    "using Microsoft.SemanticKernel.AI.Embeddings.VectorOperations;"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The notebook uses:\n",
+    "\n",
+    "* **text-davinci-003** to generate a random image description\n",
+    "* **OpenAI Dall-E 2** to transform the image description into an image\n",
+    "* **text-embedding-ada-002** to compare your guess against the real image description"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    },
+    "vscode": {
+     "languageId": "polyglot-notebook"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "var (useAzureOpenAI, model, azureEndpoint, apiKey, orgId) = Settings.LoadFromFile();\n",
+    "\n",
+    "var kernel = Microsoft.SemanticKernel.Kernel.Builder\n",
+    "    .Configure(c =>\n",
+    "    {\n",
+    "        c.AddOpenAIEmbeddingGeneration(\"ada\", \"text-embedding-ada-002\", apiKey);\n",
+    "        c.AddOpenAITextCompletion(\"davinci\", \"text-davinci-003\", apiKey, orgId);\n",
+    "        c.AddOpenAIImageGeneration(\"dallE\", apiKey, orgId);\n",
+    "    })\n",
+    "    .Build();\n",
+    "\n",
+    "var dallE = kernel.GetService<IImageGeneration>();\n",
```

Comment:
Include a short description of what's going on here and how this would work with other image generation models like from Stable Diffusion or Midjourney